tasks/89_symmertic-difference/symmertic-difference.js

```diff
@@ -0,0 +1,15 @@
+export function sym(...args) {
+  return args
+      .reduce((result, curr) => {
+        curr = [...new Set(curr)];
```

Comment:
Set has forEach method, there is no reason to transform to Set and after to array